<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>1011</width>
     <height>481</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Выражение</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Причина</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Рекомендации</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Переменная</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Значение</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Для осени</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Для весны</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="db_save_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить все</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addLineBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить правило</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeLineBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить правило</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="add_plus_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_minus_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
